### ISSUE TITLE
{hitCount} not display on HitsStats | ElasticSearch 7

### ISSUE THREAD

**Issue by Fluorz** [searchkit/searchkit#698]
Hello,

I got this bug : 

![image](https://user-images.githubusercontent.com/16563634/57680710-84cb4d00-762e-11e9-935a-775de31a1387.png)

When I change 
```        var props = {
            bemBlocks: this.bemBlocks,
            translate: this.translate,
            timeTaken: timeTaken,
            hitsCount: hitsCount,
            resultsFoundLabel: this.translate("hitstats.results_found", {
                timeTaken: timeTaken,
                hitCount: hitsCount
            })
```

For 

```        var props = {
            bemBlocks: this.bemBlocks,
            translate: this.translate,
            timeTaken: timeTaken,
            hitsCount: hitsCount,
            resultsFoundLabel: this.translate("hitstats.results_found", {
                timeTaken: timeTaken,
                hitCount: hitsCount.value
            })
```
On HitsStats.tsx line : 75

My HitCount Object : 

![image](https://user-images.githubusercontent.com/16563634/57681123-5ef27800-762f-11e9-8ece-2920e3464a13.png)

I opened a pull request #697 , but it seems that your test is not running with elastic search 7

**Comment by flts** [comment]
For the time being a way to make it work in Elasticsearch 7 is to add rest_total_hits_as_int=true as a request parameter in the search request. This will make the total value a integer again instead of an object type.

https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#_literal_hits_total_literal_is_now_an_object_in_the_search_response

Or newer versions of Searchkit will only work for newer versions of Elasticsearch or the library needs a way to define the ES version its talking to. It all depends in which direction the owners of the library want it to go.

**Comment by tatejones** [comment]
What is the suggest technique for setting this?  Do you have to create a new queryBuilder function to replace the SimpleQueryString

**Comment by flts** [comment]
Well this additional parameter is set as an Url Query. So no need for a new QueryBuilder.
So at the place where you give the full Url to the ES server and the index you also add the additional url query parameter.

**Comment by Fluorz** [comment]
It would be nice if we could adapt to the version of ElasticSearch.

I also changed Filtered Query to adapt to ES 7.0
https://www.elastic.co/guide/en/elasticsearch/reference/5.5/query-dsl-filtered-query.html

``
export function FilteredQuery(filtered:FilteredQueryOptions){
  return {filtered}
}
``

For

``
export function FilteredQuery(bool:FilteredQueryOptions){
  return {bool}
}
``

On packages/searchkit/src/core/query/query_dsl/compound/FilteredQuery.ts

**Comment by tatejones** [comment]
Thanks.  The following addition to my config was made and the hit count now appears.

const host = "http://localhost:9200/"
const searchkit = new SearchkitManager(host, { searchUrlPath : "_search?rest_total_hits_as_int=true"})

**Comment by joemcelroy** [comment]
fixed in v2.4.1
